WordPress/Classes/Utility/WPFeedbackGenerators.swift

```diff
@@ -0,0 +1,75 @@
+import UIKit
+
+@objc
+public enum WPNotificationFeedbackType : Int {
+    case Success
```

Comment:
As far as I know, the Swift 3 migrator will handle changing the case of enum values. I did initially have this in lowercase, Swift 3 style, but it imports that way into obj-c and looks weird (e.g. `WPNotificationFeedbackTypesuccess`). I presume once we're using the Swift 3 compiler it'll automagically use camel case in obj-c.